examples/dmd/parametric_tw_csv.cpp

```diff
@@ -10,24 +10,44 @@
 
 // Compile with: make parametric_tw_csv
 //
-// Generate CSV or HDF database on heat conduction with either
-// heat_conduction_csv.sh or heat_conduction_hdf.sh (HDF is more efficient).
+// This script performs serial or time windowing DMD on general datasets in CSV or HDF formats.
```

Comment:
I think `serial` here means without time windowing, but `serial` may be understood as "single-process". Would it be better to say "DMD with or without time windowing"?